README.md

```diff
@@ -19,7 +19,7 @@
 
 # Apache Beam
 
-[Apache Beam](http://beam.apache.org/) is a unified model for defining both batch and streaming data-parallel processing pipelines, as well as a set of language-specific SDKs for constructing pipelines and Runners for executing them on distributed processing backends, including [Apache Flink](http://flink.apache.org/), [Apache Spark](http://spark.apache.org/), [Google Cloud Dataflow](http://cloud.google.com/dataflow/), and [Hazelcast Jet](https://jet.hazelcast.org/).
```

Comment:
I understand the website is no longer valid. We can change it to https://hazelcast.com/ at the moment. For proposal to deprecate and decommission a runner support please open a GitHub issue and/or send a email thread to Beam dev list.

Reply:
sure, updated!